hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ControllerManagementTest.java

```diff
@@ -626,11 +754,32 @@ public void recoverFindOrRegisterTargetIfItDoesNotExistIsNotInvokedForOtherExcep
         }
     }
 
+    @Test
+    @Description("Retry is aborted when an unchecked exception is thrown and the exception should also be "
+            + "rethrown")
+    public void recoverFindOrRegisterTargetIfItDoesNotExistWithNameIsNotInvokedForOtherExceptions() {
```

Comment:
remove this test, because it does not make sense due to `findOrRegisterTargetIfItDoesNotExist` overloading

Reply:
Removed.